lib/whatsapp_sdk/api/phone_numbers.rb

```diff
@@ -40,6 +40,53 @@ def registered_number(phone_number_id)
           data_class_type: WhatsappSdk::Api::Responses::PhoneNumberDataResponse
         )
       end
+
+      # Register a phone number.
+      #
+      # @param phone_number_id [Integer] The registered number we want to retrieve.
+      # @param params [Hash] Params to update.
+      # @return [WhatsappSdk::Api::Response] Response object.
+      sig do
+        params(
+          phone_number_id: Integer,
+          params: T::Hash[T.untyped, T.untyped]
+        ).returns(WhatsappSdk::Api::Response)
+      end
+      def register_number(phone_number_id, **params)
+        response = send_request(
+          http_method: "post",
+          endpoint: "#{phone_number_id}/register",
+          params: { messaging_product: 'whatsapp' }.merge(params)
+        )
+
+        WhatsappSdk::Api::Response.new(
+          response: response,
+          data_class_type: WhatsappSdk::Api::Responses::PhoneNumberDataResponse
+        )
+      end
+
+      # Deregister a phone number.
+      #
+      # @param phone_number_id [Integer] The registered number we want to retrieve.
+      # @return [WhatsappSdk::Api::Response] Response object.
+      sig do
+        params(
+          phone_number_id: Integer,
+          params: T::Hash[T.untyped, T.untyped]
```

Comment:
`params` is not needed
```suggestion
          params: T::Hash[T.untyped, T.untyped]
```

Reply:
Done, it was removed.